### PR TITLE
Jdmartinez36 msft telephony solutions

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited description for length and to add the PBX keyword.

I found one possible way to add the search term "Skype for Business PBX", using it as a description for the Enterprise Voice (Skype for Business Server) option. I'm not sure it correctly describes the option, and the author should verify.

@DaniHalfin

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by DaniHalfin** [comment]
I'm reviewing. @scanum - any other PRs from @jdmartinez36 will be reviewed by me. Thanks.

File: Skype/SfbHybrid/hybrid/msft-telephony-solutions.md [hunk 1]
```diff
@@ -38,15 +39,15 @@ You should use this article along with the associated technical diagram, which p
...
 
 For more information, see [What is Phone System in Microsoft 365 or Office 365](https://docs.microsoft.com/MicrosoftTeams/what-is-phone-system-in-office-365).
 
 ### Enterprise Voice (Skype for Business Server)
 
-Enterprise Voice is Microsoft's technology for enabling call control and Private Branch Exchange (PBX) capabilities in the on-premises Skype for Business Server. This option can only be connected to the Public Switched Telephone Network by using your own telephony carrier. 
+Enterprise Voice is Microsoft's technology for enabling call control and Private Branch Exchange (PBX) capabilities in the on-premises Skype for Business Server. This Skype for Business PBX option can only be connected to the Public Switched Telephone Network by using your own telephony carrier.
```
Comment (SerdarSoysal) [inline Skype/SfbHybrid/hybrid/msft-telephony-solutions.md:50]: There is no such thing as "Skype for Business PBX". We should not add that phrase in this paragraph. Other than that, this PR is ok to merge.

Comment (CarolynRowe) [reply Skype/SfbHybrid/hybrid/msft-telephony-solutions.md:50]: Right.  That nomenclature would imply that Skype for Business PBX is a product name.  The paragraph already describes PBX capabilities.  We shouldn't make up names.

Comment (CarolynRowe) [reply Skype/SfbHybrid/hybrid/msft-telephony-solutions.md:50]: One other thing:  The description that was added is not technically accurate.  

"Learn about Microsoft's telephony solutions, including Private Branch Exchange (PBX) and Phone System with Calling Plan options."

Phone System is the PBX.   Phone System can be combined with Calling Plan options or with Direct Routing for PSTN connectivity options.

**Comment by CarolynRowe** [comment]
This description is not accurate:  "Learn about Microsoft's telephony solutions, including Private Branch Exchange (PBX) and Phone System with Calling Plan options."

Please change to:  Learn about Microsoft's telephony solutions, including Phone System, which provides Private Branch Exchange (PBX) functionality and PSTN connectivity options, which include Calling Plans and Direct Routing.

**Comment by CarolynRowe** [comment]
Fixing punctuation!

Please change to: Learn about Microsoft's telephony solutions, including Phone System--which provides Private Branch Exchange (PBX) functionality--and PSTN connectivity options, which include Calling Plans and Direct Routing.

**Comment by jdmartinez36** [comment]
Hi @CarolynRowe 

Google cuts off the description at 160 characters in search results. Would it be alright to shorten it to: "Learn about Microsoft's telephony solutions, including Phone System--which provides Private Branch Exchange (PBX) functionality--and PSTN connectivity options."?

**Comment by SerdarSoysal** [comment]
@jdmartinez36 / @CarolynRowe how about this:

Learn   about Microsoft's telephony solutions: Phone System (Private Branch Exchange   - PBX) and PSTN connectivity options (Calling Plans and Direct Routing).

**Comment by CarolynRowe** [comment]
Perfect!  Let's go with Serdar's modifications.

**Comment by jdmartinez36** [comment]
@CarolynRowe @SerdarSoysal 

Thanks for the feedback! I've committed the new description and removed the "PBX" from "Skype for Business PBX..." per your review. If everything looks good, I'll let Dani know this is complete.

**Comment by jdmartinez36** [comment]
Hi @DaniHalfin 

I've integrated the edits requested by Serdar and Carolyn.

**Pull request merged**